src/harness/reference_models/fss_purge/testdata/fss_record_0.json

```diff
@@ -0,0 +1,29 @@
+{
+  "id": "incumbent/ibfs/0",
+  "type": "FSS",
+  "deploymentParam": 
+  [
+    {
+      "installationParam": {
+        "latitude": 38.985278,
+        "longitude":  -77.445833,
+        "height": 26.5,
+        "elevation": 100,
+        "heightType": "AGL",
+        "antennaGain": 0,
+        "antennaBeamwidth": 0,
+        "antennaAzimuth": 270,
+        "antennaDowntilt": 0,
+        "indoorDeployment": true,
```

Comment:
The indoorDeployment flag should not be part of the fss_record

Reply:
Thanks for catching this. It has   been removed.